README.md

```diff
@@ -1,20 +1,50 @@
-## Getting Started
+<p align="center">
+  <img src="public/logo.svg" alt="Trent Computer Science Club Association Logo" width="200">
+</p>
 
-This project is a simple starting point for any next.js applications. It is configured to use `tailwind css`, `typescript`, `next.js`, `eslint` and `prettier`.
+# Trent Computer Science Club Association Website
 
-In order to get started, fork this project, clone the forked repository and run `npm install`, then run `npm run dev` to start the development server.
+This repository contains the source code for the official website of the Trent University Computer Science Club Association. The website serves as a central hub for club activities, announcements, resources, and information about our club's events.
+
+## Pages
+
+- **[Home](https://tcsca.ca)** - The main page for the website
+- **[About Us](https://tcsca.ca/About)** - The page for our about us page
+- **[Events](https://tcsca.ca/Events)** - The page for our events
+- **[Contact Us](https://tcsca.ca/Contact)** - The page for our contact us page
+
+## Layout and Structure
+
+- **[pages](pages)** - This folder contains the source code for each page with `_app.tsx` and `_document.tsx` being special files that control global page content.
+- **[components](components)** - This folder contains all components for the project
+- **[layouts](layouts)** - This folder contains all layouts for the project
+- **[styles](styles)** - This folder contains all styles for the project
+  - **[components](styles/components)** - This folder contains all styles for the components
+  - **[layouts](styles/layouts)** - This folder contains all styles for the layouts
+- **[utils.ts](utils.ts)** - This file contains all of our global utility functions
+
+- Additionally, in the root directly you will find a `config.ts` file and `config.yaml` file.
+  - `config.ts` is the backbone of our site and ensures type safety for our configuration
+  - `config.yaml` is the configuration file for the website
+
+## License
```

Comment:
can we remove this part I think on a project like this license should stay in the license file.